LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
This should be modified to the Apache license, not removed

Reply:
Fixed in [7799a7a](https://github.com/warp-ds/core/pull/3/commits/7799a7abe0a4fee433b5ebf201ce6a69887445d3) and added a "license" entry to `package.json` in [ba4a93d](https://github.com/warp-ds/core/pull/3/commits/ba4a93ded42e80e50707db960325a6c3c9ff7d37). 

We also noticed that the [license in warp drive](https://github.com/warp-ds/drive/blob/master/package.json#L31) was `"MIT"`, but maybe it should also be `"Apache-2.0"`. Should we update it?

Reply:
Yes update please!